src/pydvl/value/shapley/__init__.py

```diff
@@ -23,15 +22,17 @@
     permutation_exact_shapley,
 )
 from pydvl.value.shapley.types import ShapleyMode
+from pydvl.value.stopping import MaxUpdates, StoppingCriterion
 
 __all__ = ["compute_shapley_values"]
 
 
 def compute_shapley_values(
```

Comment:
There are still a few inconsistencies in this function, mostly wrt. stopping and extra kwargs. But it's ok-ish for now I think.